### PR TITLE
Announce Software Citation Workshop

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
Update centre column and add an annoucement

### CODE REVIEW

File: _layouts/main.html [hunk 1]
```diff
@@ -62,15 +62,22 @@ <h2>Meetings</h2>
...
-      Click <a href="https://github.com/issues?q=is%3Aissue+is%3Aopen+archived%3Afalse+sort%3Aupdated-desc+label%3Ahacktoberfest+org%3Ahsf-training+org%3Ascikit-hep+org%3Ahsf">here</a> to find all eligible issues!
+      How do we cite software in HEP and how do we give people proper recognition? It's 
+      a vital question for our community.
     </p>
-    <img src="{{ '/images/Hfest-Logo-2-Color-Void@2x.png' | relative_url }}"
-    alt="Hacktoberfest-logo" style="display: block; width:100%; margin-left: auto; margin-right: auto;">
-   </div>
+    <p>
+      The HSF and IRIS-HEP are organising a (virtual) mini-workshop to discuss just that,
+      with discussion on current practices from the experiments,
```
Comment (hegner) [inline _layouts/main.html:72]: not sure whether discussion should be singular or plural here?

File: _layouts/main.html [hunk 2]
```diff
@@ -62,15 +62,22 @@ <h2>Meetings</h2>
...
+      the view from developers, and also input from Zenodo and Inspire.
+    </p>
+    <p>
+      Register now
+      for the <a href="https://indico.cern.ch/event/1211229/">workshop</a> to
+      give your input and help us decide on what we believe are best practices to follow.
```
Comment (hegner) [inline _layouts/main.html:78]: that sounds odd. "Help us to decide what we believe". What about "help us to find out on which practices would be most useful to bring our field and community forward". Only much later after the process we can start to "believe".

File: announcements/_posts/2022-11-02-software-citation.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+title: Software Citation and Recognition Workshop, 22-23 November 2022
+author: Graeme Stewart
+layout: newsletter
+symbol: glyphicon-calendar
+link: https://indico.cern.ch/event/1211229/
+until: 2022-11-24
+---
+
+We are pleased to hold a mini-workshop event on the topic of “Software Citation and Recognition in HEP”, which will be held virtually over two 1/2 days on November 22-23, 2022.
```
Comment (hegner) [inline announcements/_posts/2022-11-02-software-citation.md:10]: "hold ... held" is a duplication. "We are pleased to announce..."

File: announcements/_posts/2022-11-02-software-citation.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+This meeting aims to provide a community discussion around ways in which HEP experiments handle citation of software and recognition for software efforts that enable physics results disseminated to the public.
```
Comment (hegner) [inline announcements/_posts/2022-11-02-software-citation.md:12]: maybe more forward looking? So what about "best handle"?

**Review by hegner — CHANGES_REQUESTED** [review]

File: _layouts/main.html (hunk 1 above)
Comment (graeme-a-stewart) [reply _layouts/main.html:72]: I think singular is best, but it could be either

**Review by hegner — APPROVED** [review]

**Pull request merged**